docs/content/en/docs/concepts/metrics/_index.md

```diff
@@ -13,56 +13,63 @@ A `KeptnMetric` is a CRD representing a metric.
 The metric will be collected from the provider specified in the
 specs.provider.name field.
 The query is a string in the provider-specific query language, used to obtain a metric.
-Providing the metrics as CRD into a K8s cluster will facilitate the reusability of this data across multiple components.
-Furthermore, this allows using multiple observability platforms for different metrics.
+Providing the metrics as CRDs in a K8s cluster will facilitate the reusability of this data across multiple components.
+Furthermore, this allows using multiple observability platforms for different metrics at the same time.
```

Comment:
Do you want to note that, at this time, only one occurence of each observability platform is allowed per namespace or is that overkill?

Reply:
No, with multi provider support that limitation is gone. You can now specify multiple of the same type :)